assets/launcher/lang/sklmessages_es_MX.properties

```diff
@@ -70,36 +70,36 @@ about.translation.authors=SrRapero720
 
 # Login Screen
 login.button.type.microsoft=Acceder con Microsoft
-login.button.type.offline=Acceder Offline
+login.button.type.offline=Iniciar Sesion
 login.label.username=Usuario
-login.dropdown.name=Usuarios con Session activa
+login.dropdown.name=Sesiones activas
 login.button.play=Jugar
 login.button.logout=Cerrar session
-login.button.switch.offline=Cambiar a modo Offline
-login.button.switch.online=Cambiar a modo Online
+login.button.switch.offline=Modo offline (sin cuenta)
+login.button.switch.online=Modo Online (microsoft)
```

Comment:
Keep the original meaning of the string, do not add any additional brackets or your own explanation of the string